index.html

```diff
@@ -0,0 +1 @@
+<html><head><title>Task robot admin page</title><link rel="stylesheet" href="/style.css"/><meta name="viewport" content="width=device-width, initial-scale=1"/></head><body class="wrapper"><div class="row"><div class="column"><h2>Activity pages</h2><h3><a href="/survey">Do survey</a></h3><h3> <a href="/results">Show results</a></h3><p> <a href="/results/json">as JSON</a></p></div></div></body></html>
```

Comment:
Is this a fallback?

Reply:
I had to do a revert  when developing when debugging, so there's a commit that reverted everything back. So it seems like alot of things changed and nothing much changed because of the revert.

The race conditions are prevented because the set_collections just sets the string.
```
  set_collection(collection: string): void {
    this.collection = collection
  }
```
Then when you call a function like find, then it use the set up function, so all transactions occur when function call is invoked.

```
  async set_up() {
    await this.client.connect()
    const database = this.client.db(this.db)
    const collection_obj = database.collection(this.collection)
    return collection_obj
  }

  async find(query: any) {
    const collection_obj = await this.set_up()
...
```

Taking a second look. This seems not very clean.
How about we change the api to ```async find(query: any) TO async find(query: any, collection: string) ```

Reply:
Not sure what that index.html is doing. I'm pretty sure I can delete it in the next round if you like the api change.

Reply:
Yes, I prefer that API. Let's swap it over before merging.